.pre-commit-config.yaml

```diff
@@ -105,10 +105,10 @@ repos:
           - '--show-fixes'
 
   - repo: https://github.com/teemtee/tmt.git
-    rev: 1.28.2
+    rev: 1.29.0
     hooks:
     - id: tmt-lint
       additional_dependencies:
-        # Make sure we use fmf compatible with tmt in the repo, not the 1.26.0 version.
+        # Make sure we use fmf compatible with tmt in the repo, not the 1.29.0 version.
```

Comment:
Is this still valid? Doesn't 1.29 have fmf>=1.3.0 already?

Reply:
Right you are! So the explict `fmf` dep is no more needed.